marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/MarkLogicRepositoryConnection.java

```diff
@@ -429,7 +429,7 @@ public MarkLogicUpdateQuery prepareUpdate(String queryString,String baseURI) thr
      */
     @Override
     public MarkLogicUpdateQuery prepareUpdate(QueryLanguage queryLanguage, String queryString) throws RepositoryException, MalformedQueryException {
-       return prepareUpdate(queryLanguage, queryString, null);
+        return prepareUpdate(queryLanguage, queryString, null);
```

Comment:
I know it's not a big deal, but often fixing whitespace issues like this makes a PR look cleaner.